### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by lovelyserafima**

### CODE REVIEW

File: src/components/functionalComponent.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+const helloWorldComponent = (props) => (<h1>Hello World in functional component</h1>);
```
Comment (yanaSelin) [inline src/components/functionalComponent.js]: Functional components still need to be started with upper case letter. Also you can omit props here, but leave it function, I mean `() => (<h1>Hello World in functional component</h1>);`

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import ReactDOM from "react-dom";
+import element from "./components/reactCreateElementComponent";
+import ReactComponent from "./components/reactComponent";
+import ReactPureComponent from "./components/reactPureComponent";
+import functionalComponent from "./components/functionalComponent";
+
+ReactDOM.render(element, document.getElementById('React.createElement'));
+ReactDOM.render(<ReactComponent/>, document.getElementById('React.Component'));
+ReactDOM.render(<ReactPureComponent/>, document.getElementById('React.PureComponent'));
+ReactDOM.render(functionalComponent, document.getElementById("Functional component"));
```
Comment (yanaSelin) [inline src/index.js]: The same, functional component name should be started with uppercase letter and passed as jsx

File: dist/index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+<head>
+    <meta charset="UTF-8">
+    <title>Hello world messages</title>
+</head>
+<body>
+<div id="React.createElement"></div>
+<div id="React.Component"></div>
+<div id="React.PureComponent"></div>
+<div id="Functional component"></div>
+<script type="text/javascript" src="index_bundle.js"></script></body>
```
[the remark this answers is not in the document]
Comment (lovelyserafima) [reply dist/index.html]: done

File: server/server.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+let express = require('express');
+let app = express();
```
[the remark this answers is not in the document]
Comment (lovelyserafima) [reply server/server.js]: done

File: server/server.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+
+app.get('/', function (req, res) {
+    res.send('Hello World!');
```
[the remark this answers is not in the document]
Comment (lovelyserafima) [reply server/server.js]: done

File: src/components/functionalComponent.js (hunk 1 above)
Comment (lovelyserafima) [reply src/components/functionalComponent.js]: done

File: src/index.js (hunk 2 above)
Comment (lovelyserafima) [reply src/index.js]: done

**Pull request merged**